### ISSUE TITLE
Asset files fail to play on iOS with 'AVPlayerItem.Status.failed' using latest master branch

### ISSUE THREAD

**Issue by rmorrin** [luanpotter/audioplayers#879]
When using the latest version of the master branch (due there not yet being a new version with a fix for #843) `AudioCache` fails to play any local asset sounds on iOS.

**Code to Reproduce**
The issue is present in the repo's own example code ([here](https://github.com/luanpotter/audioplayers/tree/master/packages/audioplayers/example)). But a minimal reproducible example is as follows:

**pubspec.yaml:**
```yaml
...

dependencies:
  flutter:
    sdk: flutter
  audioplayers:
      git:
          url: git://github.com/luanpotter/audioplayers.git
          ref: master
          path: packages/audioplayers

flutter:
  assets:
    - assets/bell.mp3
```

**main.dart:**
```dart
import 'package:flutter/material.dart';
import 'package:audioplayers/audioplayers.dart';

void main() {
  AudioPlayer.logEnabled = true;
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Audioplayers Repro',
      home: Scaffold(
        body: Center(
          child: TextButton(
            child: Text('Play Sound'),
            onPressed: () {
              final player = AudioCache();
              player.play('bell.mp3');
            },
          ),
        ),
      ),
    );
  }
}

```

**Log Errors**
```
flutter: _platformCallHandler call audio.onError {value: AVPlayerItem.Status.failed, playerId: 8f884c5e-eeb6-4f5b-adca-b602481ffd74}
flutter: _platformCallHandler call audio.onError {value: AVPlayerItem.Status.failed, playerId: c0f86748-1320-4e10-8d5e-14beeee2795c}
```

**Files/URLs/Sources**
Repo with full minimal reproducible example available at: https://github.com/rmorrin/audioplayers-asset-repro

**Platforms**
This is occurring on iOS only, tested on both simulator and a real device. Android is working as expected.

* OS: iOS
* OS version: iOS 14.5
* Device: Occurs on both iOS simulator as well as a real device
* flutter version: [`flutter doctor -v` output](https://gist.github.com/rmorrin/a0235771073fe366d799df2334ae3003)
* audioplayers version: Latest master branch @ 836f7f3

**Comment by AhmedNourJamalElDin** [comment]
+1

**Comment by jacksonw765** [comment]
Having the same problem as well.

**Comment by luanpotter** [comment]
I fixed that an released 0.19, please lmk if the problem persists! :)

**Comment by jacksonw765** [comment]
It works, thank you.

**Comment by rmorrin** [comment]
@luanpotter can confirm this is resolved by updating to the 0.19 release. Thanks! 😁

**Comment by AhmedNourJamalElDin** [comment]
Did you guys get it working with Flutter v2 and audioplayers v0.19.0?

because after upgrading flutter, I'm getting bunch of errors, something like these:

```
flutter: _platformCallHandler call audio.onSeekComplete {value: true, playerId: xyz}
flutter: _platformCallHandler call audio.onDuration {value: 2328, playerId: abc}
```

However it was working well in Flutter v1.

P.S. I'm cachine the files by URL and passing isLocal = true but I tried isLocal = false too.

![image](https://user-images.githubusercontent.com/18269514/118676527-27150100-b804-11eb-9073-77f9e54dec37.png)
